src/components/FormValidator.js

```diff
@@ -66,7 +66,7 @@ class FormValidator {
   enableValidation() {
     this._form.addEventListener("submit", (event) => {
       event.preventDefault();
-      this._disableSubmitButton();
+      this.disableSubmitButton();
```

Comment:
Please, delete `this.disableSubmitButton();` from line 69

You need to **disable** the button only in a `then` block after a **successful response** to be able to click it again if there is a server error.